argilla/src/argilla/datasets/_io/_hub.py

```diff
@@ -164,13 +165,18 @@ def from_hub(
                     client=client,
                     with_records=with_records,
                     settings=settings,
+                    split=split,
```

Comment:
I think we should also add the incoming **kwargs, right?
```suggestion
                    split=split,
                    **kwargs
```